operations/operations/company-processes/publishing/publishing-guidelines/confidentiality-guidelines.md

```diff
@@ -5,6 +5,7 @@ By default, Mattermost defaults to [open actions](../../../../../company/about-m
 Below is a set of guidelines on what information is considered confidential and should not be published publicly:
 
 * Financial information related to Mattermost, such as revenue and cash, as this may impact potential future funding and IPO.
+* Use metrics and analysis related to Mattermost deployments, such as telemetry, trial conversions, DAU, as this may impact potential future funding and IPO.
```

Comment:
```suggestion
* Usage metrics and analysis related to Mattermost deployments, such as telemetry, trial conversions, DAU, as this may impact potential future funding and IPO.
```

Reply:
Thanks!